### ISSUE TITLE
Spirit Sheet

### ISSUE THREAD

**Issue by DeHophran** [clevett/Roll20-character-sheet-Shadowrun-5th-Edition#3]
Hi there,

not sure if this is the correct why to make a feature request, I just signed up to GitHub for that.

Have you ever thought about adding a Spirit form to the sheet?

Idealy with a field, where you enter the force of the spirit and this then automatically adds to the Attributes and Initiatives, or they also for the rolls take this field into account?

And also include the skills and Spirit Powers in the sheet? 

Thanks. :)

**Comment by clevett** [comment]
I've considered adding a few of the common spirit powers to handle the math. Particularly for Immunity.

I wont automatically calculate attributes of the spirit based on the Force. For two main reasons. 1) There are a lot of exceptions all over SR5 content which are a pain to account for. 2) It would be violating Catalyst's copywrite and Roll20's rules about character creation in sheet automation. While CGL would probably never raise a fuss and Roll20 hasn't taken action against other character sheets that do it, I prefer to respect the boundaries.

I tried a few years ago to get CGL to get on Roll20 officially then I'd have been more willing to support these sort fo features. CGL never signed the paperwork. _shrug_

TL:DR: There are a LOT of spirits in SR5. I have no interested in doing that much data entry unpaid and unsanction by copywrite holder.

**Comment by DeHophran** [comment]
Hello clevett,

thanks for getting back to me. I understand your reasoning behind your decision,. By now I also worked it out with the Grunt template in your character sheet. Having only 1 field to fill in with a value would be very convenient, but your character sheet is already a powerful tool. This force field was my main wish in this request, the skills and abilities I just mentioned because why not, even as I was expecting that this may possibly cause some copyright problems. 

Thank you for all the effort you put into creating this character sheet. It really is powerful and easy to use. :)
